chapters/install/os/via-pip.rst

```diff
@@ -51,14 +45,14 @@ For Docker users the following command updates Jina in the Docker container:
    docker pull jinaai/jina
 
 
-Install from the master branch
+Install Jina development or pre-release
```

Comment:
Alternative: `Install Jina Devel (Pre-release) version - for developers to test their local changes on Jina's latest master branch`

Reply:
Ok, since you, me and Alex have different views, let's see what @nan-wang thinks? :)